uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -363,8 +363,11 @@ sub runHrrtInsertion {
     $command .= " -verbose " if ($this->{'verbose'});
 
     my $output = $this->runCommandWithExitCode($command);
-
-    return $output;
+    
+    if ( $output == 0 ) {
```

Comment:
error detection is made with
`if ( !$output ) {` and success code (0) is a falsy value.